frontend/src/components/Bill/Bill.jsx

```diff
@@ -1,186 +1,164 @@
 import { Icon } from '@iconify/react';
-import { PropTypes } from 'prop-types';
+import PropTypes from 'prop-types';
 import { useState, useEffect } from 'react';
 import { useParams } from 'react-router-dom';
 
 import LocForm from './LocForm';
 import './Bill.css';
 
-function getSponsorNumber(billData) {
-  const {
-    activeVersion,
-    amendments: { items },
-  } = billData;
-  return (
-    1 +
-    items[activeVersion].coSponsors.size +
-    items[activeVersion].multiSponsors.size
-  );
-}
+// Helper functions
+const getSponsorNumber = (billData) => {
+  const { activeVersion, amendments: { items } } = billData;
+  return 1 + items[activeVersion].coSponsors.size + items[activeVersion].multiSponsors.size;
+};
```

Comment:
This is not an actual change, just a formatting change. Can you go into your setting and find the value for this (check both the user & workspace settings):

<img width="891" alt="Screenshot 2024-10-02 at 1 48 00 PM" src="https://github.com/user-attachments/assets/f9f4af68-6f61-43b3-9abc-e4f8769dc318">

Also, the `defaultFormatter` for `javascriptreact`?

Reply:
There's so much formatting changes here that it's hard to parse what the actual changes are.

Reply:
I removed some of the unnecessary formatting changes. Also, my print width is 80, is that okay?